src/irr/asset/CImageWriterTGA.h

```diff
@@ -28,7 +28,7 @@ class CImageWriterTGA : public asset::IAssetWriter
         return ext;
     }
 
-    virtual uint64_t getSupportedAssetTypesBitfield() const override { return asset::IAsset::ET_IMAGE_VIEW; }
+    virtual uint64_t getSupportedAssetTypesBitfield() const override { return asset::IAsset::ET_IMAGE; }
```

Comment:
no you want them to save image views, I know its not symmetrical but creating a view of an image is the only way to choose which layer+mipmap gets saved to file for a limited export such as JPG